app/templates/admin/createEvent.html

```diff
@@ -26,62 +26,65 @@
 {% set eventPast = event.isPastStart if not isNewEvent else False %}
 {% set tabName = 'edit' %}
 
-
 {% block scripts %}
-  {{super()}}
+  {{super()}} 
+  <!-- Include jQuery -->
+  <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+  <!-- Include jQuery UI (required for datepicker) -->
+  <link rel="stylesheet" href="https://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
+  <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
+  <!-- Include jQuery Timepicker -->
+  <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/timepicker/1.3.5/jquery.timepicker.min.css">
+  <script src="//cdnjs.cloudflare.com/ajax/libs/timepicker/1.3.5/jquery.timepicker.min.js"></script>
+  <!-- Custom Styles -->
+  <link href="https://cdn.jsdelivr.net/gh/kartik-v/bootstrap-fileinput@5.5.0/css/fileinput.min.css" media="all" rel="stylesheet" type="text/css" />
```

Comment:
This stylesheet doesn't need to be in the scripts block.

Reply:
moved into style block